experiments/train_se3_diffusion.py

```diff
@@ -122,7 +138,7 @@ def __init__(
         self._optimizer = torch.optim.Adam(
             self._model.parameters(), lr=self._exp_conf.learning_rate)
         if ckpt_opt is not None:
-            self._optimizer.load_state_dict(ckpt_opt, strict=True)
+            self._optimizer.load_state_dict(ckpt_opt)
```

Comment:
What's the reason for removing strict? I quite like having True as the default. I once had a bug where I didn't set strict=True, the weights were never loaded and no error was given...